app/javascript/components/App.js

```diff
@@ -13,19 +12,42 @@ import Dash from "./components/Dash";
 import LandingPage from "./components/LandingPage";
 import Header from "./components/Header";
 import YourTrips from "./pages/YourTrips";
+import CreateSight from "./pages/CreateSight";
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      trips: [],
+      trips: false,
+      businesses: false,
     };
   }
 
   componentDidMount() {
     this.indexTrips();
   }
 
+  yelpApi = (item, location) => {
```

Comment:
```suggestion
  searchSights = (item, location) => {
```
Because the front-end is making a call to your backend, the front-end shouldn't know where that data is actually coming from. As far as the front-end is concerned, it's getting data from the backend. It's best to conceal this knowledge (because front-end logic can get exposed).

Reply:
like rename the method?